src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -63,12 +71,15 @@ if ($verbose) {
     if (!(Test-Path $logDir)) {
         mkdir $logDir
     }
-    $logFile = Join-Path $logDir 'admin-api-util.log'
+    $timestamp = Get-Date -Format "yyyyMMddTHHmmssffffZ"

```

Comment:
Would you write some documentation how to enable the logging for end users so that they can troubleshoot by themselves?